vagrant/ClearLinux.yml

```diff
@@ -0,0 +1,15 @@
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2019 Intel Corporation
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+qat_driver_pkgs:
+  - linux-lts2018-dev
```

Comment:
why this kernel?

Reply:
This vagrant image came with LTS version